toolsWebsite/tools/templates/tools/checkout.html

```diff
@@ -0,0 +1,101 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="{% static 'tools/styles.css' %}" type="text/css"/>
+    <script src="https://use.fontawesome.com/fbcbee1ea9.js"></script>
+    <title>Projects</title>
+  </head>
+
+  <body class="d-flex flex-column">
+    <!-- Navbar Start -->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+      <div class="container">
+        <a class="navbar-brand" href="/tools/">Tool Co-op</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ml-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="/tools/">All Tools <span class="sr-only">(current)</span></a>
+            </li>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Tools
+              </a>
+              <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                <a class="dropdown-item" href="/tools/available">Available</a>
+                <a class="dropdown-item" href="/tools/checked_out">Checked Out</a>
+                <a class="dropdown-item" href="/tools/overdue">Overdue</a>
+              </div>
+            </li>
+            <li class="nav-item active">
+              <a class="nav-link" href="/tools/project">Projects</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">History</a>
+            </li>
+            <li class="nav-item">
+              <ul class="navbar-nav ml-auto ml-md-0">
+                <li class="nav-item dropdown no-arrow">
+                  <a class="nav-link dropdown-toggle" href="#" id="cartDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    <i class="fa fa-cart-arrow-down"></i>
+                  </a>
+                  <div class="dropdown-menu dropdown-menu-right" aria-labelledby="cartDropdown">
+                    <div id="cartDropdownItems"></div>
+                    <a href="/checkout">
+                      <button class="btn btn-outline-info my-2 my-sm-0 right shoppingDropdownButton">
+                        Check Out
+                      </button>
+                    </a>
+                  </div>
+                </li>
```

Comment:
We should be able to add the tools that are going to be checked out in this `li` so that they appear in the dialog that pops up when you click the shopping cart drop down.

Reply:
Not specifically on this html page, but on all pages with the navbar header

Reply:
Sounds good.

Reply:
Made all the navbars have this functionality.  Doesn't seem that dry but we might be able to figure out a way to create a navbar template of some sort.